engines/gob/inter_v7.cpp

```diff
@@ -263,6 +352,218 @@ void Inter_v7::o7_loadFunctions() {
 	_vm->_game->loadFunctions(tot, flags);
 }
 
+void Inter_v7::copyFile(const Common::String &sourceFile, const Common::String &destFile) {
+	SaveLoad::SaveMode mode1 = _vm->_saveLoad->getSaveMode(sourceFile.c_str());
+	SaveLoad::SaveMode mode2 = _vm->_saveLoad->getSaveMode(destFile.c_str());
+
+	if (mode2 == SaveLoad::kSaveModeIgnore || mode2 == SaveLoad::kSaveModeExists)
+		return;
+	else if (mode2 == SaveLoad::kSaveModeSave) {
+		if (mode1 == SaveLoad::kSaveModeNone) {
+			Common::SeekableReadStream *stream = _vm->_dataIO->getFile(sourceFile);
+			if (!stream)
+				return;
+
+			int32 size = stream->size();
+			byte *data = new byte[size];
+			stream->read(data, size);
+
+			_vm->_saveLoad->saveFromRaw(destFile.c_str(), data, size, 0);
```

Comment:
maybe  a `TODO` comment should get added here to implement a `Stream` version of `saveFromRaw` - e.g. `saveFromStream` to work around this memory allocation.